mephisto/providers/mock/wrap_crowd_source.js

```diff
@@ -53,3 +54,22 @@ function handleSubmitToProvider(task_data) {
     alert("The task has been submitted! Data: " + JSON.stringify(task_data))
     return true;
 }
+
+// Adding event listener instead of using window.onerror prevents the error to be caught twice
+window.addEventListener('error', function (event) {
```

Comment:
@pringshia is there a reason this listener is defined in `wrap_crowd_source.js` and not as a part of `mephisto-task`?

Reply:
the first place I put it inside `mephisto-task` but Pratik said it's better to be inside each provider separately but I didn't find it convenience because we will need to put it in each Provider's `wrap_crowd_source.js`, @pringshia  could you elaborate why is this better?

Reply:
@JackUrb - The reason for putting this in `wrap_crowd_source.js` as opposed to `mephisto-task` is so that this global catch-all error handling can also apply to:

- Errors in static HTML tasks
- Errors that may occur at the React library level, before the application code has had a chance to run (maybe related due to a build linking / webpack error?)